v22.1/aws-dms.md

```diff
@@ -5,27 +5,33 @@ toc: true
 docs_area: develop
 ---
 
-This page has instructions for setting up [AWS DMS](https://aws.amazon.com/dms/) to migrate data to CockroachDB from an existing, publicly-hosted database containing application data such as MySQL, Oracle, or PostgreSQL.
+This page has instructions for setting up [AWS Database Migration Service (DMS)](https://aws.amazon.com/dms/) to migrate data to CockroachDB from an existing, publicly-hosted database containing application data such as MySQL, Oracle, or PostgreSQL.
 
 For a detailed tutorial about using AWS DMS and information about specific migration tasks, see the [AWS DMS documentation site](https://docs.aws.amazon.com/dms/latest/userguide/Welcome.html).
 
-We have tested AWS DMS with CockroachDB as a target enough to claim **preview-level** support. If you encounter problems in CockroachDB, please [open an issue](https://github.com/cockroachdb/cockroach/issues/new) with details to help us make progress toward full support.
+We have tested AWS DMS with CockroachDB as a target enough to claim **preview-level** support. If you encounter problems in CockroachDB, [open an issue](https://github.com/cockroachdb/cockroach/issues/new) with details to help us make progress toward full support.
```

Comment:
If AWS DMS is in preview, should this page also have the Preview include?

Reply:
Also, this might be a me thing, but "claim" here sounds forceful, maybe? What about "... provides **preview-level** support."